can_tools/scrapers/nytimes/nyt_cases_deaths.py

```diff
@@ -0,0 +1,275 @@
+    ("48187", "2020-09-21", 1587),  # Guadalupe County, TX
+    ("48201", "2020-09-21", 13622),  # Harris County, TX
+    ("48239", "2020-09-21", 77),  # Jackson County, TX
+    ("48255", "2020-09-21", 181),  # Karnes County, TX
+    ("48265", "2020-09-21", 142),  # Kerr County, TX
+    ("48271", "2020-09-21", 19),  # Kinney County, TX
+    ("48285", "2020-09-21", 252),  # Lavaca County, TX
+    ("48355", "2020-09-21", 231),  # Nueces County, TX
+    ("48385", "2020-09-21", 12),  # Real County, TX
+    ("48493", "2020-09-21", 307),  # Wilson County, TX
+    ("48507", "2020-09-21", 52),  # Zavala County, TX
+    ("48407", "2020-09-22", 1),  # San Jacinto, TX
+    # https://trello.com/c/GKUHUbyK/483-2020-10-03-collin-county-tx-spike-on-daily-new-cases
+    ("48201", "2020-10-02", 2438),  # Harris County, TX
+    ("48085", "2020-10-03", 1202),  # Collin County, TX
+]
+
+STATE_BACKFILLED_CASES = [
+    # On 2020-07-24, CT reported a backfill of 440 additional positive cases.
+    # https://portal.ct.gov/Office-of-the-Governor/News/Press-Releases/2020/07-2020/Governor-Lamont-Coronavirus-Update-July-24
+    ("09", "2020-07-24", 440),
+    # https://portal.ct.gov/Office-of-the-Governor/News/Press-Releases/2020/07-2020/Governor-Lamont-Coronavirus-Update-July-29
+    ("09", "2020-07-29", 384),
+    # http://www.floridahealth.gov/newsroom/2020/09/090120-1112-covid19.pr.html
+    ("12", "2020-09-01", 3870),
+    # https://twitter.com/DHSWI/status/1301980028209713153?s=20
+    # This number is a bit fuzzy, but on 9/3 there were 767 cases.  There were 1400 cases
+    # on 9/4 including backfill.  Estimating 800 cases on 9/4.
+    ("55", "2020-09-04", 1400 - 800),
+    # https://directorsblog.health.azdhs.gov/covid-19-antigen-tests/
+    ("04", "2020-09-17", 577),
+    # https://twitter.com/AZDHS/status/1306974258124468226
+    ("04", "2020-09-18", 764),
+    # https://trello.com/c/9EbBRV1Z/472-2020-09-26-north-carolina-change-in-methodology-to-count-cases
+    ("37", "2020-09-25", 4544),
+    # https://www.al.com/news/2020/10/alabama-adds-3852-covid-cases-after-influx-of-backlogged-data-dating-back-to-june.html
+    ("01", "2020-10-23", 2565),
+]
+
+
+class NYTimesCasesDeaths(FederalDashboard, ETagCacheMixin):
+    source_name = "The New York Times"
+    source = "https://github.com/nytimes/covid-19-data"
+
+    has_location = True
+    provider: str = "nyt"
+    location_type = ""  # multiple location types, so we identify the type with a column
+    cache_file = "nyt_cases_deaths.txt"
+    file_slugs = {
+        "county": "us-counties.csv",
+        "state": "us-states.csv",
+        "nation": "us.csv",
+    }
+
+    variables = {
+        "cases": CUMULATIVE_CASES_PEOPLE,
+        "deaths": CUMULATIVE_DEATHS_PEOPLE,
+    }
+
+    def __init__(self, execution_dt: pd.Timestamp = pd.Timestamp.utcnow()):
+        # All the files appear to be updated at the same time.
+        # Use the national file for cache checking since it's the smallest
```

Comment:
Hrm. I'm not sure if we should count on this.  It's probably true for the daily deploy but if they post mid-day updates I'm not sure it'll be true, and I could see this causing difficult-to-debug confusion down the road.

[Here](https://github.com/nytimes/covid-19-data/commit/d50443c374d8b614cc386667a0a63d11fe77740f)'s an example where /us-counties.csv was updated but /us.csv was not.

Maybe we should just not bother doing caching for NYT?  Or will that cause meaningful problems you think?  Otherwise I guess we have to turn it into 3 different scrapers or come up with a different way to do caching?

Reply:
ah that's true, good catch. 

Initially (in a9070ecb0828968dae8f7a64ef7f3de0a7176f30), I had these split up into separate scrapers, but I merged them together because the backfill code modifies both the state and county data.

Let's try this without caching for the time being, and if that does cause problems we can revisit.